taxi_service/urls.py

```diff
@@ -14,8 +14,11 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("taxi/", include("taxi.urls", namespace="taxi"))
```

Comment:
Your default path should lead to the `urls` from the `taxi` app, change this line
```suggestion
    path("", include("taxi.urls", namespace="taxi"))
```